code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -7,6 +7,8 @@
 	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
 	use_skintones = 1
 	skinned_type = /obj/item/stack/sheet/animalhide/human
+	disliked_food = GROSS | RAW

```

Comment:
Kinda dumb that humans can't eat raw food. Have you never had carpaccio or prosciutto?